pkg/actors/actors.go

```diff
@@ -1220,7 +1217,7 @@ func (a *actorsRuntime) ExecuteLocalOrRemoteActorReminder(ctx context.Context, r
 		}
 		a.lock.Unlock()
 		go func() {
-			log.Debugf("Deleting reminder which was cancelled: %s", reminder.Key())
+			log.Errorf("Deleting reminder which was cancelled: %s", reminder.Key())
```

Comment:
Could this not cause log explosion?

Reply:
Yes, thanks. Good catch.

Was using for testing.